content/using-atom/sections/github-package.md

```diff
@@ -11,13 +11,20 @@ The github package brings Git and GitHub integration right inside Atom.
 - [Stage](#stage)
 - [Discard](#discard-changes)
 - [Commit](#commit)
-- [Amend](#amend-previous-commit)
-- [Push](#push)
-- [Pull](#pull)
+- [Amend and undo](#amend-and-undo)
+- [Publish and push](#publish-and-push)
+- [Fetch and pull](#fetch-and-pull)
 - [Resolve conflicts](#resolve-conflicts)
+- [Create a Pull Request](#create-a-pull-request)
 - [View Pull Requests](#view-pull-requests)
+- [Checkout a Pull Request](#checkout-a-pull-request)
+- [Open any Issue or Pull Request](#open-any-issue-or-pull-request)
 
-Most of the functionality lives in the Git and GitHub panel. There are different ways to access them, probably the most common way is through their keybindings:
+Most of the functionality lives in the Git and GitHub panel.
+
+![The Git and GitHub panels](../../images/github-panels.png "The Git and GitHub panels")
+
+There are different ways to access them, probably the most common way is through their keybindings:
```

Comment:
> There are different ways to access them, including keybindings:

It might be worth mentioning the menu option path as well? Packages -> GitHub -> Toggle Git Tab and Toggle GitHub Tab.